src/cdk/testing/component-harness.ts

```diff
@@ -375,6 +375,31 @@ export abstract class ComponentHarness {
   }
 }
 
+
+/**
+ * Base class for component harnesses that authors should extend if they anticipate that consumers
+ * of the harness may want to access other harnesses within the component.
+ */
+export abstract class UserContentComponentHarness extends ComponentHarness
```

Comment:
How about `ContentContainerComponentHarness`? Or even just `ContainerComponentHarness`

Reply:
I had a similar idea when I was talking to @mmalerba about it (`WrapperComponentHarness`), but I think I like Miles' proposal more, because it's more explicit about it being targeted towards projected content.

Reply:
Jeremy convinced me that we should go with `ContentContainerComponentHarness`, since `UserContentComponentHarness` could be interpreted as a harness for the user content